### PR TITLE
PyTorch Mixed Precision Examples

### PR DESCRIPTION
**Pull request by zhen-xie**
PyTorch Mixed Precision Examples For CompPerfWorkshop

### CODE REVIEW

File: 10_reduced-precision/PyTorch/README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Auotmatic Mixed Precision in PyTorch
 
+-----------
+Training in FP16 that is in half precision results in slightly faster training in nVidia cards that supports half precision ops. Also the memory requirements of the models weights are almost halved since we use 16-bit format to store the weights instead of 32-bits.
```
Comment (felker) [inline 10_reduced-precision/PyTorch/README.md:4]: nVidia ---> NVIDIA

File: 10_reduced-precision/PyTorch/README.md [hunk 2]
```diff
@@ -33,5 +36,34 @@ Essentially, the only difference is: instead of wrapping the optimizer object wi
 scaling optimizer object, PyTorch has the user explicitly call the adaptive loss scale
 update from a separate object.
 
-Refer to the [PyTorch AMP
-Recipe](https://pytorch.org/tutorials/recipes/recipes/amp_recipe.html) for a detailed walkthrough.
+
+We will use the 2022 release NVIDIA-optimiized NGC Singularity container for
+PyTorch 3.x in this walkthrough:
+```
+singularity exec --nv -B /lus /lus/theta-fs0/projects/datascience/zhen/docker/pytorch_22.01-py3.sif bash
```
Comment (felker) [inline 10_reduced-precision/PyTorch/README.md]: switch to a container in `/lus/theta-fs0/software/thetagpu/nvidia-containers/pytorch/`

**Comment by felker** [comment]
LGTM. I would omit the `__pycache__` directory and the `.pyc` files from version control

File: 10_reduced-precision/PyTorch/README.md (hunk 2 above)
Comment (zhen-xie) [reply 10_reduced-precision/PyTorch/README.md]: Can you help me to download the pytorch-22.01-py3 to this directory?

**Comment by zhen-xie** [comment]
will delete __pycache__ directory and the .pyc files

**Pull request merged**